src/ontology/agro-edit.owl

```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010163> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010163> "2022-01-04T00:43:04Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010163> "farm woodland"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/AGRO_00010163> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_00000078> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00000109>)))
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010164> (scattered trees)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://besjournals.onlinelibrary.wiley.com/doi/10.1111/1365-2664.12943") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010164> "Trees which have ecological functions in landscapes like offering shelter and food sources for animals, enhancing landscape connectivity by acting as stepping stones, and functioning as nuclei for plant regeneration in disturbed landscapes."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010164> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010164> "2022-01-04T00:50:58Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/AGRO_00010164> "isolated trees"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/AGRO_00010164> "paddock trees"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/AGRO_00010164> "pasture trees"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/AGRO_00010164> "remnant trees"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/AGRO_00010164> "Scattered trees may enhance the provision of ecosystem services that might benefit farmers and owners of rural properties, such as pollination of crops, shading for cattle, regulation of nitrogen dynamics and carbon sequestration, herbaceous production, and wood provision."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010164> "scattered trees"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010164> <http://purl.obolibrary.org/obo/AGRO_00000585>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010165> (taungya farming)
```

Comment:
Could you please import the PATO term : http://purl.obolibrary.org/obo/PATO_0000083 ## phase and move the cropping phase of taungya term under this PATO term?

Reply:
Added issue for parent class 'Phase' import (#169)
Updated label and parent class to imported class. 
Commit af1d207ba51d4f8b07d0abed9ed89af9c1e10e98